integration/commands_authentication_test.go

```diff
@@ -27,12 +33,55 @@ import (
 func TestCommandsAuthenticationLogout(t *testing.T) {
```

Comment:
Logout does not fail on authentication check, if user is not authenticated, it just returns success.